### PR TITLE
CLOUDP-219151: fix template for create azure/gcp privateEndpoints

### PR DESCRIPTION
**Pull request by tibulca**
## Proposed changes
fix template for create azure/gcp privateEndpoints

_Jira ticket:_ CLOUDP-219151

### CODE REVIEW

File: internal/cli/atlas/privateendpoints/azure/create.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func (opts *CreateOpts) initStore(ctx context.Context) func() error {
 	}
 }
 
-var createTemplate = "Private endpoint '{{.ID}}' created.\n"
+var createTemplate = "Private endpoint '{{.Id}}' created.\n"
```
Comment (tibulca) [inline internal/cli/atlas/privateendpoints/azure/create.go:46]: the fix

**Review by andreaangiolillo — DISMISSED** [review]
LGTM

**Review by andreaangiolillo — APPROVED** [review]

File: internal/cli/atlas/privateendpoints/gcp/create.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ func (opts *CreateOpts) initStore(ctx context.Context) func() error {
 	}
 }
 
-var createTemplate = "GCP private endpoint '{{.ID}}' created.\n"
+var createTemplate = "GCP private endpoint '{{.Id}}' created.\n"
```
Comment (jwilliams-mongo) [inline internal/cli/atlas/privateendpoints/gcp/create.go:46]: it doesn't appear as if we include the cloud provider in the message for the other providers. we should probably omit it here to remain consistent.
```suggestion
var createTemplate = "Private endpoint '{{.Id}}' created.\n"
```

**Review by jwilliams-mongo — APPROVED** [review]
@tibulca LGTM % one minor comment. We don't specify a cloud provider in the templates for AWS and Azure, so I think we can omit it for GCP. Conversely we can add it for the other two

**Review by andreaangiolillo — APPROVED** [review]

**Pull request merged**